Source/Build Specs/Common gRPC Types.vipb

```diff
@@ -0,0 +1,391 @@
+<VI_Package_Builder_Settings Version="2020.1" Created_Date="2023-01-23 14:05:21" Modified_Date="2023-11-15 14:16:56" Creator="lvadmin" Comments="" ID="2ad2328419eeb300544b1df2aa0dc54a">
+  <Library_General_Settings>
+    <Package_File_Name>ni_protobuf_types</Package_File_Name>
```

Comment:
> [@dixonjoel](https://github.com/dixonjoel) pushed 1 commit.
>
>https://github.com/ni/measurementlink-labview/commit/509ee59ddb81ef91d0e630740debd6a250b294c5 Add build spec for ni_common_types-x.x.x.x.vip

Is the package name `ni_common_types` or `ni_protobuf_types`?

Why don't the product name and package file name correspond to each other?

Reply:
It is ni_protobuf_types. I used 'Common' and 'gRPC' in a couple places that are user-facing since I didn't know how well customers would understand that. But we can standardize on 'Protobuf' perhaps. How about these?
* NI Protobuf Types.vipb
* ni_protobuf_types-x.x.x.x.vip
* Product Name: NI Protobuf Types
* Description: Protobuf types for use across NI products

Anything else that I missed?

Reply:
> I used 'Common' and 'gRPC' in a couple places that are user-facing since I didn't know how well customers would understand that.

I see. 

You could expand "Protobuf" to "Protocol Buffers" in the English name/description, but I don't know how much that helps.

Another approach is to lean on ni-apis and use some variation on "NI API message types" 

> Description: Protobuf types for use across NI products

Ideas for clarifying this:
- "for use across" -> "used by" or "shared by"?
- "NI products" -> "NI APIs" or "NI gRPC APIs"?
- "types" -> "data types" or "message types"?

Reply:
I would personally try to stick to a convention based naming scheme for a lot of this stuff in terms of the file names and visible package names. If there are user facing descriptions / names within the contents of the vipb/vipm files, then using friendlier naming / descriptions at that point seems fine. While that may initially lead to package names that seem less friendly, I think it will make it easier to navigate/browse for packages in the long term once you understand the packaging convention. Hopefully, the VIPM store has sophisticated enough search that users can still find what they are looking for if they make search queries using English phrases rather than exact product names.

Reply:
Package name is currently ni_protobuf_types-<version>.vip. So I think I'm following this. For the .vipb name, I currently have 'NI gRPC Types.vipb'. Should I change that to, e.g. ni_protobuf_types.vipb perhaps?

Reply:
I think keeping them the same would be less confusing.